lib/rust/parser/src/syntax/tree.rs

```diff
@@ -851,6 +852,8 @@ impl From<SyntaxError> for Cow<'static, str> {
             StmtInvalidAssignmentOrMethod => "Invalid assignment or method definition",
             StmtLhsInvalidOperatorSpacing =>
                 "Each operator on the left side of an assignment operator must be applied to two operands, with the same spacing on each side",
+            StmtUnexpectedAssignmentInModuleBody =>
+                "Unexpected variable assignment in module statement",
```

Comment:
How could one make variable assignment in module body? Aren't such assignments automatically interpreted as functions without parameters?

Reply:
> Aren't such assignments automatically interpreted as functions without parameters?

Only if the LHS is a (qualified) name; if it's another type of pattern, (e.g. a destructuring binding like "[a, b] ="), it's treated as an assignment even in a context where an assignment is an error.